<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StopInterface</class>
 <widget class="QWidget" name="StopInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>停机数据分析</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,2,2" columnminimumwidth="0,0,0">
   <item row="0" column="2">
    <widget class="CardWidget" name="CardWidget_2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="SubtitleLabel" name="SubtitleLabel_4">
        <property name="text">
         <string>停机前钢卷信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TableView" name="TableView_2"/>
      </item>
      <item>
       <widget class="SubtitleLabel" name="SubtitleLabel_5">
        <property name="text">
         <string>停机后钢卷信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TableView" name="TableView_3"/>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="PushButton" name="mergeOnlineTablePushButton">
        <property name="text">
         <string>合并酸轧判定</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
          <normaloff>:/icons/icons/format.png</normaloff>:/icons/icons/format.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="ProgressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="CardWidget" name="CardWidget">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="SubtitleLabel" name="SubtitleLabel">
          <property name="text">
           <string>数据预览窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="TableView" name="TableView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="PushButton" name="importRawDataPushButton">
          <property name="text">
           <string>导入并格式化</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/导入.png</normaloff>:/icons/icons/导入.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PushButton" name="concatDataPushButton">
          <property name="text">
           <string>合并</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/数据集合并.png</normaloff>:/icons/icons/数据集合并.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PushButton" name="exportPushButton">
          <property name="text">
           <string>导出</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/导出.png</normaloff>:/icons/icons/导出.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CardWidget" name="CardWidget_3">
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,0">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="SubtitleLabel" name="SubtitleLabel_2">
           <property name="text">
            <string>整年停机原因统计</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ComboBox" name="ComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>年份</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="SubtitleLabel" name="SubtitleLabel_3">
           <property name="text">
            <string>停机前后IU均值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="PushButton" name="importProcessDataPushButton">
        <property name="text">
         <string>导入处理数据</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
          <normaloff>:/icons/icons/导入.png</normaloff>:/icons/icons/导入.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
